exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlWorker.java

```diff
@@ -183,4 +184,23 @@ private static PhysicalPlan getQueryPlan(QueryContext context, String sql, Point
 
     return handler.getPlan(sqlNode);
   }
+
+  //Wrap with Auto Limit
+  private static SqlNode wrapWithAutoLimit(SqlConverter parser, QueryContext context, String sql) {
+    SqlNode sqlNode = parser.parse(sql);
+    if (context.isAutoLimitEnabled() && sqlNode.getKind().belongsTo(SqlKind.QUERY)) {
+      int autoLimitRowCount = context.getAutoLimitRowCount();
```

Comment:
The StringBuilder code within ```if(){ ... }``` may be simplified to : 
```java
      String wrappedSql = String.format("-- [autoLimit: %2$d rows]%nSELECT * FROM (%n %1$s %n) LIMIT %2$d",
          sql, context.getAutoLimitRowCount());
      logger.debug("Query text to execute: {}", wrappedSql);
      sqlNode = parser.parse(wrappedSql);
```

Reply:
+1